book/src/platformer/01_introduction.md

```diff
@@ -0,0 +1,13 @@
+# Learn agb part II: Platformer
+
+In this section you'll learn how to use `agb` and Tiled to make a platformer.
```

Comment:
```suggestion
In this section you'll learn how to use `agb` and [Tiled](https://www.mapeditor.org/) to make a platformer.
```

Feels worth a link to Tiled here because it isn't on the second one.